src/components/Player/Pages/Counts/Counts.jsx

```diff
@@ -13,14 +14,18 @@ import styles from './Counts.css';
 const Counts = ({ playerId, counts, getPlayerCounts }) => (
   <div>
     <TableFilterForm submitAction={getPlayerCounts} id={playerId} page="counts" />
-    <div className={styles.countsContainer}>
-      {Object.keys(counts).map((key, index) => (
-        <div key={index} className={styles.countTable}>
-          <TableContainer title={deSnake(key)} />
-          <Table paginated columns={playerCountsColumns} data={counts[key].list} />
-        </div>
-      ))}
-    </div>
+    <Row>
+      <div className={styles.countsContainer}>
+        {Object.keys(counts).map((key, index) => (
+          <Col xs={12} sm={12} md={6} lg={4} key={index}>
+            <div className={styles.countTable}>
+              <TableContainer title={deSnake(key)} />
+              <Table paginated columns={playerCountsColumns} data={counts[key].list} />
```

Comment:
Pending these changes I will pass this off as it seems to be pretty straight forward and easy and won't require a large amount of effort if we abstract the row/columns away to table container.